README.md

```diff
@@ -60,4 +57,70 @@ These tables are an addition to the existing PEtab format to describe the covari
 
 - No table
 
-  If no table is provided, only the variances, but no covariances are estimated.
\ No newline at end of file
+  If no table is provided, only the variances, but no covariances are estimated.
+  
+## Measurement table
+
+In addition to the existing descriptions of fixed effects, the NLME extension adds the following columns:
+
+- `$covariateId` one column per distinct covariate
+  
+## Covariate table
+
+This table describes the addition of covariate table in the PEtab NLME extension.
+
+### Detailed field description
+
+- `covariateId` [STRING]
+
+   ID of the covariate.
+
+- `covariateParameter` [STRING]
+
+   Defines the parameter where the covariate effect (COVEFF) is modelled. It must be a `parameterId` defined in the parameter table. The COVEFF would be multiplied with the `parameterId`.
+
+-  `covariateFormula` [STRING: 'exclude' or 'fractional' or 'power' or free text formula]
+
+   Covariate effect function as plain text formula expression. The user can define its own formula or can use predefined models: 'exclude' or 'fractional' or 'power'.
+   - 'exclude' --> Covariate not included on the parameter.
+   $\text{COVEFF}=1$
+
+   - 'fractional' --> Linear function. For categorical and continuous covariates.
+   
+   For continuous covariates:
+   $\text{COVEFF}= 1 + \theta_{cov} \times \left(\text{covariateId} - median(\text{covariateId})\right)$
```

Comment:
Sorry I did not explain this in the README, quickly $\theta_{cov}$ represents the magnitude / effect of the covariate on a specific parameter. Usually needs to be estimated, e.g. "estimated 30% decrease in the drug clearance in female vs male"

I will correct this in the README accordingly 👍🏼

Reply:
OK. But how about including $\theta_{cov}$ in the `covariateFormula`, and following the convention that everything in the formula must be either

- a covariate as specified in the measurement table
- a valid SBML ID (e.g. a parameter or species)
- a parameter defined in the parameter table.

This would mean, that Instead of `power`, you would have to do `power^theta_cov`, and put `theta_cov` in the parameter table, which would give us `lowerBound`, `upperBound`, `nominalValue`, `estimate`, `objectivePrior`, etc. for free.

Which also tells me that my interpretation of `lowerBound`,  `upperBound` and `nominalValue` in the covariate table was probably wrong. Now I think that they refer to $\theta_{cov}$. Is this correct? If so, we could either

- get rid of them in the covariate table
- still use `lowerBound` and `upperBound`, but with the meaning I suggested earlier (i.e. just for verification purposes).